llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -201,6 +196,10 @@ cl::opt<module_split::IRSplitMode> SplitMode(
 cl::opt<bool> DoSymGen{"symbols", cl::desc("generate exported symbol files"),
                        cl::cat(PostLinkCat)};
 
+cl::opt<bool> SkipPropsGen{"skip-properties-gen",
+                           cl::desc("do not generate module properties files"),
+                           cl::cat(PostLinkCat)};
+
```

Comment:
I suggest we invert the logic and add `properties` option (similar to `symbols`) which will enable generation of properties.

Reply:
yeah i originally did it inverted to minimize requires changes to existing tests, but i agree it is more consistent. done in latest commit.

Reply:
What does the -properties option accomplish? In the current implementation, I see that this option just controls the generation of .props file. is that accurate? Thanks

Reply:
I hate to suggest this (especially after Nick having modified all the tests to add -properties). I would have preferred '-skip-properties' and -skip-symbols' as they seem to be enabled by default and adding -properties and -symbols seems extraneous. But it's just a nit. 

Thanks

Reply:
I think we are moving towards flipping the default. Once we move from `sycl-post-link` to `clang-linker-wrapper`, this option won't be needed.

Reply:
agree, in the (near) future we will not need this options and can call these from `clang-linker-wrapper`.